awx_collection/plugins/lookup/schedule_rruleset.py

```diff
@@ -210,10 +210,10 @@ def process_integer(self, field_name, rule, min_value, max_value, rule_number):
 
     def process_list(self, field_name, rule, valid_list, rule_number):
         return_values = []
-        if isinstance(rule[field_name], list):
+        if isinstance(rule[field_name], str):
             rule[field_name] = rule[field_name].split(',')
         for value in rule[field_name]:
-            value = value.strip()
+            value = value.strip().lower()
```

Comment:
Agree. Thanks for this fix and associated test. Hopefully I can get the other patch merged, then this rebased on top of it.